Source/RomanNumeral.c

```diff
@@ -13,4 +13,24 @@ static char * Append(char *output, const char *toAppend)
 
 void RomanNumeral_Convert(unsigned input, char *output)
 {
+   char *CharactersArray []= {"M", "CM", "D", "CD", "C", "XC", "L", "XL", "X", "IX", "V", "IV", "I"};
+   unsigned DecimalArray []= {1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1};
+   unsigned i, arrayAdress, tableSize;
+
+   for(i = 0; i <= 999; i++)
+      {
+         output[i] = 0;
+      }
+
+   tableSize = sizeof(CharactersArray) / sizeof(char *);
+   for(arrayAdress = 0; arrayAdress < tableSize; arrayAdress++)
```

Comment:
same as the previous for loop, you can do
```suggestion
   for(unsigned arrayAdress = 0; arrayAdress < tableSize; arrayAdress++)
```

Reply:
arrayAddress

Reply:
Done!